mmv1/third_party/terraform/services/cloudrun/resource_cloud_run_service_test.go.erb

```diff
@@ -1440,4 +1428,69 @@ resource "google_cloud_run_service" "default" {
 `, name, project)
```

Comment:
per https://github.com/hashicorp/terraform-provider-google/issues/19206#issuecomment-2315983310, also remove the lifecycle.ignore_changes directly above this line.